packages/engine/lib/flatbuffers_gen/src/batch_generated.rs

```diff
@@ -6,13 +6,14 @@
     clippy::used_underscore_binding,
```

Comment:
We really should get Clippy and rustfmt to ignore generated files

Reply:
For rustfmt at least for checking in CI, they should be ignored.